db/errors.go

```diff
@@ -7,3 +7,22 @@ var (
 	ErrInvalidData   = fmt.Errorf("invalid data provided")
 	ErrAlreadyExists = fmt.Errorf("already exists")
 )
+
+var (
```

Comment:
What is the reason for creating a new var block here?

Reply:
no reason at all, fixed